etc/behat/suites.yml

```diff
@@ -37,6 +37,7 @@ imports:
     - suites/ui/addressing.yml
     - suites/ui/promotion.yml
     - suites/ui/theme.yml
+    - suites/ui/taxation.yml
```

Comment:
I think it would be even better to organize them alphabetically, but it is task for separated PR